nifi-framework-bundle/nifi-framework/nifi-file-authorizer/src/main/java/org/apache/nifi/authorization/FileUserGroupProvider.java

```diff
@@ -670,6 +684,7 @@ private synchronized void load() throws JAXBException, IllegalStateException {
 
         if (emptyTenants) {
             populateInitialUsers(tenants);
+            populateInitialGroups(tenants);
```

Comment:
This is more of an idea than a comment.  How would you feel about adding all of the initial users as a member of each of the initial groups?  As I was testing this, I found myself wanting to add myself (the Initial Admin) to my Initial Group, because managing policies at the group level has always proven to be easier.

Reply:
I see where you're going with this, however I'm not sure if that's a good default.
Say I want initial users `Alice`, `Bob`, and `Cleo` as well as initial groups `Users` and `Admins`. 
I do not think it makes a lot of sense to add the users to both groups in this case. 

In regards to the `Initial Admin Identity` (User)  and `Initial Admin Group`  however, one could argue that it would be a good idea to add the initial admin user to the initial admin group, in case both are defined. What do you think @mosermw?

Reply:
Thanks for considering this @EndzeitBegins. I remember being unsure if there was an obviously correct approach to this.  Looking back now, I agree that adding all initial users to all initial groups is going too far.

Adding the `Initial Admin Identity` user to the `Initial Admin Group` does feel like an obvious thing to do, though.  If you can make that happen, I think it makes a lot of sense.